docs/_index.md

```diff
@@ -38,30 +38,28 @@ While `super` and its accompanying data formats are production quality, the proj
 a number of different elements of the system:
 * The [super data model](formats/zed.md) is the abstract definition of the data types and semantics
 that underlie the super-structured data formats.
-* The [super data formats](formats/_index.md) are a family of
+* The [super-structured data formats](formats/_index.md) are a family of
```

Comment:
With the exception of "super data model", I'm continuing to make all attempts to avoid using "super" as a standalone adjective more than necessary. We actually haven't been saying "super data model" anywhere else yet, so it seems appropriate to match it here with how it was described in the previous bullet.